core/src/main/java/hudson/util/jna/Kernel32Utils.java

```diff
@@ -94,8 +96,12 @@ public static void createSymbolicLink(File symlink, String target, boolean dirLi
         }
     }
 
+    /**
+     * @deprecated Use {@link Util#isSymlink} to detect symbolic links and junctions instead.
+     */
+    @Deprecated
```

Comment:
Could deprecate `createSymbolicLink` now too, right?

Reply:
Yes I will deprecate it. I think we can also deprecate `getWin32FileAttributes` because it was only used by `isJunctionOrSymlink`.